src/backend/web/routes/posts.js

```diff
@@ -1,18 +1,40 @@
 const express = require('express');
-const { getPosts } = require('../../utils/storage');
+const { getPosts, getPostsCount } = require('../../utils/storage');
 const { logger } = require('../../utils/logger');
 
 const posts = express.Router();
 
 posts.get('/', async (req, res) => {
-  let redisGuids;
   const defaultNumberOfPosts = 30;
   const capNumOfPosts = 100;
+  const page = req.query.page || 1;
+
+  let redisGuids;
```

Comment:
Let's get rid of the `redis` prefix, we don't need to leak implementation details.